packaging/amazon-linux-ami-integrated/ecs-agent.spec

```diff
@@ -159,10 +159,7 @@ required routes among its preparation steps.
 
 %build
 # each of these should build for arm and amd arch
-./scripts/get-host-certs
-./scripts/build-cni-plugins
-./scripts/build-integrated true "" true true
-./scripts/build-agent-image
+make release-agent-internal
```

Comment:
Why do we use `-internal` here (vs just `release-agent` below)?

Reply:
This target is used in internal build process where we assume cni plugins have been fetched already, build in an isolated environment and do not require fetching cni plugins. ```release-agent``` is basically ```get-cni-sources``` + ```release-agent-internal```. As such, it is a single target calling these two targets but otherwise no difference between them these two.